go.mod

```diff
@@ -25,4 +26,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.6.3
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.18.10
+replace (
+	github.com/gardener/gardener-resource-manager/api => ./api
```

Comment:
cc @timuthy

Reply:
Should be fine since `github.com/gardener/gardener-resource-manager/api` is a resolvable dependency for modules depending on GRM, https://github.com/gardener/gardener-resource-manager/pull/86#issuecomment-720948404.